utils/config.js

```diff
@@ -23,11 +24,16 @@ module.exports = {
   dateOfBirthDeploymentDate: process.env.DATE_OF_BIRTH_FEATURE_DATE || '20/04/2021',
   demarchesSimplifieesId: process.env.DEMARCHES_SIMPLIFIEES_ID,
   demarchesSimplifieesUrl: process.env.DEMARCHES_SIMPLIFIEES_URL,
+  demarchesSimplifieesAutoAcceptDepartments: process.env.DEMARCHES_SIMPLIFIEES_AUTO_ACCEPT_DEPARTMENTS || [],
+  demarchesSimplifieesAutoAcceptMessage: autoAcceptMessage,
```

Comment:
Why add the message to the config and not call it directly  where needed?

Reply:
to have on single entry point for all the config stuff